lib/ibmdb.js

```diff
@@ -420,7 +420,9 @@ IBMDB.prototype.updateOrCreate = IBMDB.prototype.save =
           }
 
           if (countData[0]['CNT'] > 0) {
-            stmt = self.buildUpdate(model, data.id, data);
+            var where = {};
+            where[id] = data[id];
```

Comment:
ummmm... you are right...
Well the logic of the specific line `data.id = info[0]['CNT'];` does not make sense to me at all. How if `id` is type of string and not auto increment `id`?  I'm on vacation at the moment. @loay @superkhau could you please work on this and fix this as well as part of this PR?
Thanks